src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx

```diff
@@ -19,12 +19,14 @@ To check if you have access to these features, you can go to the [authentication
 
 Requirements to configure these settings:
```

Comment:
It might be helpful to tease out / be more obvious with what is meant by 'Requirements.' A context-giving sentence could help out with leading the reader to your bullet points.

> The New Relic plan-type used by an organization (whether it's Standard, Pro, or Enterprise) affects how they manage their users (either manually or automated) and what can be managed (configuring/editing domain settings). 

Something like that (but worded with whatever the standard verbiage is) could make parsing the bullet list a little easier.